### PR TITLE
Preface update for cypress

### PR DESCRIPTION
**Pull request by mhoeber**
@lamagnifica   tried to simplify it a little

### CODE REVIEW

File: en_us/shared/preface.rst [hunk 1]
```diff
@@ -344,15 +345,15 @@ release-specific documentation is available on docs.edx.org.
...
+  learners are currently only guided to this resource through the courseware,
+  we encourage course teams to provide learners with links to this guide as
+  needed in course updates or discussions.
 
 * `Installing, Configuring, and Running the edX Platform`_ provides
   information about installing and using Devstack and Fullstack.
 
-* `Building and Running an Open edX Course - latest`_ includes descriptions of
-  features released after the most recent official, named release.
-
```
Comment (lamagnifica) [inline en_us/shared/preface.rst:355]: ^removed intentionally?

Comment (mhoeber) [inline en_us/shared/preface.rst:355]: yes - still there in line 341

Comment (lamagnifica) [inline en_us/shared/preface.rst:355]: line 341 is for the named release

Comment (mhoeber) [inline en_us/shared/preface.rst:355]: it's not -- the whole list is latest.

File: en_us/shared/preface.rst [hunk 2]
```diff
@@ -331,8 +331,9 @@ Resources for Open edX
 **************************
 
 Hosting providers, platform extenders, core contributors, and course staff all
-use Open edX. Starting with the Birch release of Open edX, the following
-release-specific documentation is available on docs.edx.org.  
+use Open edX. EdX provides release-specific documentation, as well as the
+latest version of all guides, for Open edX users. The following
+latest documentation is available.
```
Comment (lamagnifica) [inline en_us/shared/preface.rst]: just strike the word "latest" and all is well

**Comment by lamagnifica** [comment]
:+1: 
please also remember the edx-analytics-dashboard version

**Pull request merged**